contracts/masset/MassetManager.sol

```diff
@@ -67,42 +63,13 @@ contract MassetManager is
         uint256 bassetQuantity
     );
 
-    /**
-     * @dev Emitted when tokensReceived method is called by the bridge.
-     * @param operator         Address operator requesting the transfer.
-     * @param from             Address token holder address.
-     * @param to               Address recipient address.
-     * @param amount           uint256 amount of tokens to transfer.
-     * @param userData         Bytes extra information provided by the token holder (if any).
-     * @param operatorData     Bytes extra information provided by the operator (if any).
-     */
-    event onTokensReceivedCalled(
-        address operator,
-        address from,
-        address to,
-        uint256 amount,
-        bytes userData,
-        bytes operatorData
-    );
-
-    /**
-     * @dev Emitted when onTokensMinted method is called by the bridge.
-     * @param sender           Address of the sender.
-     * @param orderAmount      Units of the masset to redeem.
-     * @param tokenAddress     Address of the bAsset to redeem.
-     * @param userData         Address of the final recipient as ABI encoded bytes.
-     */
-    event onTokensMintedCalled(
+    event MassetTokenIntermediaryChanged(
```

Comment:
naming: i suggest to replace all names and file names instances having `MassetTokenIntermediary` with `MassetTokenTransferWithPermit` for better clarity

Reply:
missing natspec

Reply:
updated in this [commit](https://github.com/DistributedCollective/mynt/pull/17/commits/57307ba37bc62d3f003c1b59b86350579a01d092)